addons/medical_vitals/functions/fnc_updateHeartRate.sqf

```diff
@@ -22,7 +22,15 @@ params ["_unit", "_hrTargetAdjustment", "_deltaT", "_syncValue"];
 
 private _heartRate = GET_HEART_RATE(_unit);
 
-if !IN_CRDC_ARRST(_unit) then {
+if IN_CRDC_ARRST(_unit) then {
+    if (alive (_unit getVariable [QEGVAR(medical,CPR_provider), objNull])) then {
+        if (_heartRate == 0) then { _syncValue = true }; // always sync on large change
```

Comment:
```suggestion
        _syncValue = (_heartRate == 0); // always sync on large change
```

Reply:
Thinking about it more, I guess the reason you did it this way is we don't want to overwrite a `true` input value.

Reply:
yeah I felt that on start/stop of CPR it needs to update immediately,
so someone else checking pulse can see it
`_syncValue = _syncValue && {(_heartRate == 0){;`
would also work but that should be similar to what's already there